application/features/SFTP.py

```
@@ -116,10 +123,12 @@
             return
 
     def zip_generator(self, cwd, file_list):
+        logger.debug(f"SFTP: zip_generator on directory: {cwd}")
```

Ruff

Logging statement uses f-string

(G004)